tools/MarkdownConverter/Converter/MarkdownSourceConverter.cs

```diff
@@ -923,6 +944,8 @@ IEnumerable<OpenXmlElement> Literal2Elements(string literal, bool isNested, bool
         }
     }
 
+    // Note for future PR: "function_members", "format_strings_1", and "format_strings_2" are the
```

Comment:
I don't understand this comment. What's happened to CreateMultiplicationTable etc? (Did we stop *using* those a while ago, but didn't remove the code?)

Reply:
Right - I see now that the tables were removed in https://github.com/dotnet/csharpstandard/pull/584.

I suggest leaving the code here for now, but I've filed https://github.com/dotnet/csharpstandard/issues/1046 to remove it in a separate PR. I'm happy to do that as soon as this one is merged.

Reply:
Added a comment to make it easier to remember this